src/libraries/System.Reflection.MetadataLoadContext/ref/System.Reflection.MetadataLoadContext.csproj

```diff
@@ -1,9 +1,13 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;net461;$(NetFrameworkCurrent)</TargetFrameworks>
```

Comment:
Not in the package.  I'll stop commenting on these, but there seem to be a number of cases where we don't include the ref in the package yet you added a net461 config to the ref.  Let me know if I'm missing something around this.